src/ros_filter.cpp

```diff
@@ -1907,6 +1916,20 @@ namespace RobotLocalization
 
     if (getFilteredOdometryMessage(filteredPosition))
     {
+      if (!validateFilterOutput(filteredPosition))
+      {
+        ROS_ERROR_STREAM("Critical Error, NaNs were detected in the output state of the filter." <<
```

Comment:
Is this still a "critical error" if it is handled relatively gracefully?

Reply:
I want to see the ROS_ERROR, but I can update the message, removing the `Critical Error` part of it.

`